pkg/hostagent/setup.go

```diff
@@ -306,21 +317,26 @@ func (agent *HostAgent) configureContainerIfaces(metadata *md.ContainerMetadata)
 		}
 		// if no mac is assigned, set it to the default Mac.
 		if len(iface.Mac) == 0 {
+			logger.Infof("No Mac assigned, assigning default")
 			iface.HostVethName, iface.Mac, err =
 				runSetupVeth(iface.Sandbox, iface.Name, agent.config.InterfaceMtu, nil)
 			if err != nil {
+				deallocIP(iface)
```

Comment:
I think it will help to log here at info level that "runSetupVeth" has returned an (we can log the error as well) and that we are returning an error.